app/src/main/java/com/example/paetz/yacguide/RouteActivity.java

```diff
@@ -50,6 +51,15 @@ public void back(View v) {
         finish();
     }
 
+    public void showMap(View v) {
+        Uri gmmIntentUri = Uri.parse("geo:"+ _rock.getLatitude() + "," + _rock.getLongitude());
+        Intent mapIntent = new Intent(Intent.ACTION_VIEW, gmmIntentUri);
+        if (mapIntent.resolveActivity(getPackageManager()) != null) {
+            startActivity(mapIntent);
+        }
```

Comment:
If there is no map tool available, you should show an according error message (using Toast) like "Keine Karten-App verfügbar."